interface.html

```diff
@@ -95,7 +113,17 @@
                       </div>
                     </div>
                     -->
-                    <div class="file-cell">
+                    <div class="file-cell search-result-cell">
+                      <div class="file-path">
+                        Path
```

Comment:
what does `Path` display? can you post a screenshot?

Reply:
![image](https://user-images.githubusercontent.com/1763029/59528574-5a97d400-8ee7-11e9-9bb2-bd85e3bd14a7.png)